common/table.js

```diff
@@ -310,6 +310,12 @@
                         vm.pendingRowValues[valIndex][obj.index] = displayValue;
                     } else {
                         vm.rowValues[valIndex][obj.index] = displayValue;
+                        // emit aggregates loaded event for [row][column]
+                        var uniqueIndex = valIndex;
+                        if (vm.config.containerIndex) {
+                            uniqueIndex = vm.config.containerIndex + "-" + uniqueIndex;
+                        }
+                        $rootScope.$emit("aggLoaded-" + uniqueIndex, obj.index);
```

Comment:
This is just nitpicky but to be consistent with the rest of the event names let's use `agg-loaded` (or `aggergate-loaded` to be more clear) instead of camel casing the name.

You should also document this event [in this jsdoc](https://github.com/informatics-isi-edu/chaise/blob/802b60b99f85d2b7abb831ff44530d3a798438aa/common/table.js#L99) (where the rest of events are documented. )